interop/run.py

```diff
@@ -216,17 +217,29 @@ def run(self):
 if get_args().debug:
   console.setLevel(logging.DEBUG)
 else:
-  console.setLevel(logging.WARNING)
+  console.setLevel(logging.INFO)
 logger.addHandler(console)
 
+implementations = copy.deepcopy(IMPLEMENTATIONS)
```

Comment:
Why do you need this copy? Can you not modify IMPLEMENTATIONS directly?

Reply:
Probably, but I intended to have `IMPLEMENTATIONS` as a const (even if Python doesn't give us a way to enforce that).